bin/cron4rucio_datasets_daily_stats.sh

```diff
@@ -293,10 +293,10 @@ function run_spark() {
     export PYTHONPATH=$SCRIPT_DIR/../src/python:$PYTHONPATH
     spark_submit_args=(
         --master yarn
-        --conf spark.executor.memory=4g
+        --conf spark.executor.memory=8g
```

Comment:
@mrceyhun doubling the memory worries me, does the script will need more when we'll get more data. In HL-LHC we will have more data and therefore we should project and estimate scalability of this approach.

Reply:
@vkuznet I don't think it will require more memory when processed data size will getting bigger. Here is the detailed story behind this change: We were running this Spark job at early the morning, 5-6am CERN time. It worked without any problem for a couple of weeks. But recently it started to fail because of Spark task/executor failures due to resource allocation. When I run it manually in the day at 10-11am with same settings, 4g, it run without errors. I do not know the root cause of the problem, because it fails in early morning but later it runs fine. Then i had a guess that, other big Spark jobs(not ours, other users') consume so much resources in the early morning, so our Spark job cannot allocate much memory in the cluster. After I set 8g memory, everything is fine in the early morning too. Unfortunately I cannot provide more details since http://ithdp2401.cern.ch:18080/ is not working to see Spark application logs.

Reply:
@vkuznet do you think it worths to merge this PR? I can further debug the issue with 4g, but I insist to use 8g for now because i'm pretty sure Spark job will fail with 4g in the current cron time.

Reply:
yes, that's fine by me. One thing to remember though, if you'll ever need to go beyond 8GB then I would consider this as a red flag because it would be hard to find nodes with higher RAM capacity in a cluster and it will require to re-design the workflow which requires such high memory footprint.

Reply:
Thanks Valentin. Yes, 16GB is impossible in Analytix cluster, I tested. I'll further debug the issue to rollback to 4GB, added to my to-do. Merging for now..